app/views/layouts/exercise_inputs/read_only_editors/_code.html.erb

```diff
@@ -1,3 +1,3 @@
 <div>
-  <%= read_only_editor content, exercise.language.name %>
+  <%= read_only_editor content, exercise.language.highlight_mode %>
```

Comment:
Great! This is the root cause of python not being highlighted in forum, right?

Reply:
Correct!